docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -56,21 +57,40 @@ Downstream effects that are not being implemented as part of this decision:
 
     * We will implement this later since we do not believe this to be an essential feature.
 
-For easier visualization, here are all of the events we plan to implement, described from end to end:
+For easier visualization, here are all of the downstream effects we plan to port, described from end to end:
+
+Downstream effects to be implemented as part of this decision:
+**************************************************************
+ ====================================== ================================================================================================ ============================================ =============================================== ========================================================================= ====================================================================================== 
+  Downstream Effect                      Context in which it's triggered                                                                  Consumer Location                            Functions Called                                General Context for Calls                                                 Expected Result                                                                       
+ ====================================== ================================================================================================ ============================================ =============================================== ========================================================================= ====================================================================================== 
+  Grades Override                        When an exam attempt is rejected.                                                                lms/djangoapps/grades/signals.py             override_subsection_grade in api.py             When we need to override a grade from any service.                        A grade override object is created or modified in the grades service within the LMS.  
+  Undo Grades Override                   When an exam attempt is verified after previously being rejected, OR when it is deleted/reset.   lms/djangoapps/grades/signals.py             undo_override_subsection_grade in services.py   When we need to undo a grade override from any service.                   A grade override object is deleted in the grades service within the LMS.              
+  Instructor Reset Subsection            When an exam attempt is deleted/reset.                                                           lms/djangoapps/instructor/signals.py         reset_student_attempts in enrollments.py        When we need to reset a student’s state in a subsection.                  A learner's state for a subsection is reset.                                          
```

Comment:
This is nit, but every time I see this, I read it as relating to the instructor dashboard reset feature, not the instructor service. This makes me read "Instructor Mark Subsection Completed the same way, even though it has nothing to do with the instructor dashboard. Do you think we need to keep "instructor" in these names, since the other downstream effects don't contain the related service names? I think it makes it confusing.